doc/MidiPlayer.xml

```diff
@@ -1,25 +1,28 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="MidiPlayer" inherits="AudioStreamPlayer" version="3.5">
 	<brief_description>
-		[b]A MIDI player class.[/b]
+		[b]A Midi and SoundFont player class.[/b]
 	</brief_description>
 	<description>
 		The MidiPlayer class can play MIDI files. It does this using a collection of sounds called SoundFont which was originally invented in the early 1990s by Creative Labs for their SoundBlaser AWE32 sound card. The MIDI specification is even older from 1983.
-		Although these technologies old, they are still used today in music industry because they are so incredibly handy. MIDI specifically has seen continous with minimal changes since since inception. 
-		In order to use this class, the node should be added to the scene tree. Create a new [AudioStreamGenerator] via inspector in [member AudioStreamPlayer.stream] and then load a [member soundfont] with [member midi]. For instance:
+		Although these technologies are old, they are still being used today in music industry because they are incredibly handy. While SoundFont has become less relevant with the development of samlping, MIDI specifically has seen seen continous use very with minimal changes to the standard. 
+		In order to use this class, the node should be added to the scene tree. Create a new [AudioStreamGenerator] via inspector in [member AudioStreamPlayer.stream] and then load a [MidiFile] containing a SoundFont should be set to  [member soundfont] and a [MidiFile] midi song set to [member midi]. For instance:
 		[codeblock]
 		func _ready():
-		    $MidiPlayer.midi = preload("res://moonlight.mid")
-		    $MidiPlayer.soundfont = preload("res://piano.sf2")
+		    $MidiPlayer.midi = load("res://moonlight.mid")
+		    $MidiPlayer.soundfont = load("res://piano.sf2")
```

Comment:
`preload()` is recommended to use at all times, because it preloads resources at parse-time.

Reply:
Change to preload instead? 👍